environment.yml

```diff
@@ -1,8 +1,8 @@
-name: mne
+name: base
 channels:
 - defaults
 dependencies:
-- python<3.7
+- python
```

Comment:
Why don't we say `python>=3.5` here?

Reply:
In practice it always should be, but I guess we might as well